es74x-cloud/src/main/scala/tech/beshu/ror/proxy/es/clients/HighLevelClientBasedClusterAdminClient.scala

```diff
@@ -0,0 +1,279 @@
+/*
+ *     Beshu Limited all rights reserved
+ */
+package tech.beshu.ror.proxy.es.clients
+
+import monix.execution.Scheduler
+import org.elasticsearch.action.{ActionFuture, ActionListener, ActionRequest, ActionResponse, ActionType}
+import org.elasticsearch.action.admin.cluster.allocation.{ClusterAllocationExplainRequest, ClusterAllocationExplainRequestBuilder, ClusterAllocationExplainResponse}
+import org.elasticsearch.action.admin.cluster.health.{ClusterHealthRequest, ClusterHealthRequestBuilder, ClusterHealthResponse}
+import org.elasticsearch.action.admin.cluster.node.hotthreads.{NodesHotThreadsRequest, NodesHotThreadsRequestBuilder, NodesHotThreadsResponse}
+import org.elasticsearch.action.admin.cluster.node.info.{NodesInfoRequest, NodesInfoRequestBuilder, NodesInfoResponse}
+import org.elasticsearch.action.admin.cluster.node.reload.NodesReloadSecureSettingsRequestBuilder
+import org.elasticsearch.action.admin.cluster.node.stats.{NodesStatsRequest, NodesStatsRequestBuilder, NodesStatsResponse}
+import org.elasticsearch.action.admin.cluster.node.tasks.cancel.{CancelTasksRequest, CancelTasksRequestBuilder, CancelTasksResponse}
+import org.elasticsearch.action.admin.cluster.node.tasks.get.{GetTaskRequest, GetTaskRequestBuilder, GetTaskResponse}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://app.codacy.com/app/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=4591272)